app/services/iiif_manifest.rb

```diff
@@ -115,7 +115,21 @@ def manifest_metadata
         create_sidecars_for(*metadata.keys)
 
         metadata.each_with_object({}) do |(key, value), hash|
-          next unless (field = exhibit_custom_fields[key])
+          # The exhibit field for the key povides the solr field name, so we
```

Comment:
Not holding it up over this, but "provides" has a typo.

Reply:
Thanks!